### PR TITLE
docs(troubleshoot): Move upgrade troubleshooting guide

### PR DESCRIPTION
**Pull request by ksubrmnn**
Signed-off-by: Kalya Subramanian <kasubra@microsoft.com>

**Description**:
Moves upgrade troubleshooting guide to the new docs directory

**Affected area**:

- New Functionality      [ ]
- Documentation          [X]
- Install                [ ]
- Control Plane          [ ]
- CLI Tool               [ ]
- Certificate Management [ ]
- Networking             [ ]
- Metrics                [ ]
- SMI Policy             [ ]
- Security               [ ]
- Tests                  [ ]
- CI System              [ ]
- Performance            [ ]
- Other                  [ ]

Please answer the following questions with yes/no.

- Does this change contain code from or inspired by another project? If so, did you notify the maintainers and provide attribution?
No

### CODE REVIEW

**Review by michelleN — APPROVED** [review]

File: docs/content/docs/troubleshooting/upgrade.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+---
+title: "Upgrade Troubleshooting"
+description: "OSM Upgrade Troubleshooting Guide"
+type: docs
```
Comment (michelleN) [inline docs/content/docs/troubleshooting/upgrade.md:4]: Do you need `aliases: ["troubleshooting"]` in this section?

Comment (flynnduism) [reply docs/content/docs/troubleshooting/upgrade.md:4]: 👍 

Good idea to make the previous url point to the new one.

Comment (michelleN) [reply docs/content/docs/troubleshooting/upgrade.md:4]: @flynnduism - could you break down what aliases are for us? Are these the tags that folks can use for search purposes in the UI?

**Review by michelleN — APPROVED** [review]

File: docs/content/docs/troubleshooting/upgrade.md (hunk 1 above)
Comment (flynnduism) [reply docs/content/docs/troubleshooting/upgrade.md:4]: Sure thing! [Aliases](https://gohugo.io/content-management/urls/#aliases) are Hugo's way to handle content redirects.

So here, the upgrade page exists at `/troubleshooting/upgrade`. If the page had previously had a diffferent url structure (like `troubleshooting-upgrades` or something) the addition of an alias would create additional pages in thge Hugo site that redirect to this page.

```
alias: ["troubleshooting-upgrades"]
```

Taking another look at this PR I see the page is being moved into the docs structure, but there isn't really a path change so I don't think an alias is needed in this instance

**Review by SanyaKochhar — APPROVED** [review]

**Pull request merged**